index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+
+
+<head>
+
+<title>
+Lawrence's CTD portfolio
+</title>
+
+  <meta charset="UTF-8">
+  <meta name="description" content="CTD portfolio">
+  <meta name="keywords" content="HTML, CSS, JavaScript">
+  <meta name="author" content="Lawrence">
+</head>
+
+<body>
+Lawrence
```

Comment:
You probably want to wrap any text in a header, paragraph, or list tag, so this could be `<p>Lawrence</p>`.